db/migrate/20160918121902_create_users_contacts.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsersContacts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users_contacts, id: false do |t|
```

Comment:
Тут глобальная проблема эти join_table нужно называть в порядке отсортированных моделей соединяемых, тогда не надо будет явно указывать join_table `contacts_users`

```
3.3.2 Creating Join Tables for has_and_belongs_to_many Associations

If you create a has_and_belongs_to_many association, you need to explicitly create the joining table. Unless the name of the join table is explicitly specified by using the :join_table option, Active Record creates the name by using the lexical book of the class names. So a join between author and book models will give the default join table name of "authors_books" because "a" outranks "b" in lexical ordering.
```
http://guides.rubyonrails.org/association_basics.html#updating-the-schema